_episodes/10-publication.md

```diff
@@ -12,6 +12,136 @@ keypoints:
+>
+> Nielsen, M. Reinventing Discovery: The New Era of Networked Science. Princeton University Press, 2011.
+
+https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000246
+
+## Code release
+
+For computational projects, releasing your work in an open repository has parallels with publications. 
+
+<img src="https://i.imgur.com/I9EKxdF.png" alt="drawing" width="800"/> 
+
+There can be specific requirements to keep code bases and/or data private. See the section below for good and not so good reasons for keeping work private.
+
+You can release code and data associated with a research article as a series of files/folders. If your project follows the folder template introduced in a previous episode, for example:
+<img src="https://github.com/jcolomb/gintonic-blog/raw/forreview/50_figures/pulication_ready_figures/2draft-templates.png" alt="drawing" width="300"/> 
+
+> Examples of a template folder tree for a computational project. https://github.com/tonic-team/Tonic-Research-Project-Template
+
+You could bundle folders into a `.zip` file and upload it to Zenodo.
+
+### Zenodo
+
+<img src="https://i.imgur.com/SUWUTKK.png" alt="drawing" width="500"/>
+
+> Zenodo is a general-purpose open-access repository developed under the European OpenAIRE program and operated by CERN. It allows researchers to deposit research papers, data sets, research software, reports, and any other research related digital artefacts.
+>
+> Uploads to Zenodo are:
+>
+> - **Safe** — your research is stored safely for the future in CERN’s Data Centre for as long as CERN exists.
+> - **Trusted** — built and operated by CERN and OpenAIRE to ensure that everyone can join in Open Science.
+> - **Citeable** — every upload is assigned a Digital Object Identifier (DOI), to make them citable and trackable.
+No waiting time — Uploads are made available online as soon as you hit publish, and your DOI is registered within seconds.
+> - **Open or closed** — Share e.g. anonymized clinical trial data with only medical professionals via our restricted access mode.
+> - **Versioning** — Easily update your dataset with our versioning feature.
+> - **GitHub integration** — Easily preserve your GitHub repository in Zenodo.
+> - **Usage statistics** — All uploads display standards compliant usage statistics
+
```

Comment:
Do we want to add in instructions about how to link github and zenodo? or maybe a link? 

This link looks good - https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content